lib/src/screens/session/disclosure/widgets/disclosure_permission_choices_screen.dart

```diff
@@ -150,6 +150,23 @@ class DisclosurePermissionChoicesScreen extends StatelessWidget {
               ),
             ),
             SizedBox(height: theme.defaultSpacing),
+            if (state is DisclosurePermissionChoicesOverview &&
+                (state as DisclosurePermissionChoicesOverview).isSignatureSession) ...[
```

Comment:
If you say `final state = this.state;` at the beginning of the function, then you don't need the type cast anymore. The compiler can derive it then.